scripts/pi-hole/php/header.php

```diff
@@ -337,34 +330,31 @@ function pidofFTL()
                         } else {
                             $pistatus = null;
                         }
-                        if ($pistatus === "1") {
+                        if ($pistatus === "53") {
                             echo '<span id="status"><i class="fa fa-circle text-green-light"></i> Active</span>';
                         } elseif ($pistatus === "0") {
                             echo '<span id="status"><i class="fa fa-circle text-red"></i> Offline</span>';
                         } elseif ($pistatus === "-1") {
                             echo '<span id="status"><i class="fa fa-circle text-red"></i> DNS service not running</span>';
+                        } elseif ($pistatus === "-2" || is_null($pistatus)) {
+                            echo '<span id="status"><i class="fa fa-circle text-red"></i> Unknown</span>';
                         } else {
-                            echo '<span id="status"><i class="fa fa-circle text-orange"></i> Unknown</span>';
+                            echo '<span id="status"><i class="fa fa-circle text-orange"></i> DNS service on port '.$pistatus.'</span>';
                         }
-
-                        // CPU Temp
-                        if($FTL)
-                        {
-                            if ($celsius >= -273.15) {
-                                echo "<span id=\"temperature\"><i class=\"fa fa-fire ";
-                                if ($celsius > $temperaturelimit) {
-                                    echo "text-red";
-                                }
-                                else
-                                {
-                                    echo "text-vivid-blue";
-                                }
-                                ?>"></i> Temp:&nbsp;<span id="rawtemp" hidden><?php echo $celsius;?></span><span id="tempdisplay"></span></span><?php
+                        ?>
+                      <br/>
```

Comment:
There seem to have snuck an extra linebreak in, that leaves a gap in the ui.
![image](https://user-images.githubusercontent.com/2525536/147459075-ef7ef47c-bf2b-456e-a1ea-121220f1e2c8.png)
![image](https://user-images.githubusercontent.com/2525536/147459127-652f565d-81da-4be6-8cdf-b5471297bee9.png)

Reply:
Thanks for your comment. The issue here is not the new linebreak, but that Pi-hole was unable to find/read a temp sensor on your machine. `$celsius` is therefore set to `-273,16` and the following if condition is never met. I added an else condition which should bring back the fire symbole with the small  notice. Please try again after `pihole -up`

Reply:
Looking good 👍

Reply:
Actually, you seem to have removed the `[pidofFTL](https://github.com/pi-hole/AdminLTE/blob/193fb7fcb9daff08eae5b48ab9895dd35bff4233/scripts/pi-hole/php/header.php#L153)` function in the `header.php` script, which sets the `$FTL` variable that's later used to show info about pihole-FTL in the `[Settings.php](https://github.com/pi-hole/AdminLTE/blob/193fb7fcb9daff08eae5b48ab9895dd35bff4233/settings.php#L235)` script. My System settings tab reports FTL is not running with this branch.

Reply:
Good catch!
I forgot that this is used in Settings.php. Re-added it. Will push later after addressing DL6ER's comment.